File: CAMPUSMONK/L5_lowerORupper.py
a = "Bibhakar"

# p=""     #lower to upper
# for i in range(0,len(a)):
#     if ord(a[i])>=ord('a') and ord(a[i])<=ord('z'):
#           p+=chr(ord(a[i])-32)
#     else:
#         p+=a[i]        
# print(p)
    #-----------------------
# o=[]   #lower to upper
# for i in a:
#     if i>='a' and i<='z':
#         o.append(chr(ord(i)-32))
#     else:
#         o.append(i)
# print(''.join(o))
    #---------------------------
o=[]   # upper TO LOWR
for i in a:
    if i>='A' and i<='Z':
        o.append(chr(ord(i)+32))
    else:
        o.append(i)
print(''.join(o))
#----------------------------
print()
print(a.upper())
print(a.lower())

File: CAMPUSMONK/L5_lowerORupper.cpp
// #include<iostream>
// #include<algorithm>
// #include<string>
// #include<cctype>
#include<bits/stdc++.h>
using namespace std;
int main()
{
    // lower upper case
    // string a = "abc";
    // string b = "ABC";
    // cout<<int('A');
    // if (a==b)
    //      cout<<"yes";
    string a= "Bibhakar";
    for(int i=0;i<a.size();i++)
    {
        // if(a[i]>='a' && a[i]<='z')
        // {
        //     a[i]-=32;
        // }
        //
        // a[i] = tolower(a[i]);  // BIG TO SMALL CONVERTER
        a[i] = toupper(a[i]);  // SMALL TO BIG CONVERTER
    }
    cout<<a<<endl;
}